crypto.py

```diff
@@ -0,0 +1,51 @@
+import requests
+import re
+
+from decorators      import is_command
+from crypto_data     import crypto_data
+
+def get_key(value):
+    for key, value in crypto_data.items():
+        if value == value:
+            return key
+
+    return "BTC"
+
+@is_command("Check all the crypto prices.")
```

Comment:
This shouldn't be decorated with @is_command as it's not callable directly from chat. The decorator on commands.crypto is correct, but as this is a sub-function is shouldn't be decorated and should really have a leading underscore on the function name.